kinder/ci/workflows/skew-1.16-on-1.15.yaml

```diff
@@ -0,0 +1,136 @@
+version: 1
+summary: |
+  This workflow tests the proper functioning of kubeadm version v1.16 with Kubernetes v1.15
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-1-16-on-1-15
+  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-X-on-Y.yaml
+  config    > https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml
+vars:
+  kubeadmVersion: "{{ resolve `ci/latest-1.16` }}"
+  kubernetesVersion: "{{ resolve `ci/latest-1.15` }}"
+  controlPlaneNodes: 3
+  workerNodes: 2
+  baseImage: kindest/base:v20190403-1ebf15f
+  image: kindest/node:test
+  clusterName: kinder-xony
+tasks:
+# Important! this worflow is using locally defined task instead of the shared skew-x-on-y-tasks.yaml
```

Comment:
[3] skew-1.16-on-1.15.yaml and skew-1.15-on-1.14.yaml are now using the same set of task; this can be refactored in a separated file to be imported by both

I'm ok also in doing this in a separated PR, but in this case please log an issue to keep track of this

Reply:
we can just consolidate this in the next cycle.